regen-network/regen-web
# feat: APP-140 add a popup with basic info about offchain project pages for users

## Description

https://regennetwork.atlassian.net/browse/APP-140?atlOrigin=eyJpIjoiZjJkNjE5NTllY2NjNDdmMWFmZDA4MjU2YWE1MTEyMGMiLCJwIjoiaiJ9



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



 From https://deploy-preview-2374--regen-marketplace.netlify.app/
1. Log in with a web2 account and create a project (either from list a project top nav button or from the create project button on your profile page), you should see the popup on top of the basic info form
2. Log in with a web3 account that is an issuer and create a project:
   - if you choose a credit class then you'll get to the basic info form without the popup being displayed
   - if you do no choose any credit class then you'll get to the basic info form with the popup on top of it, as in 1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
